packages/website/src/components/navigation/search-drawer.tsx

```diff
@@ -46,13 +46,21 @@ const remapChild = (pages: Pages, docId: string, type: string) =>
     parent: docId,
   }));
 
-// @ts-ignore
-const newDocs = data.docs.map(({ id, pagePath, children }) => ({
-  title: id,
-  pages: children
-    ? [...remapChild(children || [], id, 'nested-docs')]
-    : [{ id: 'readme', title: id, path: pagePath, type: 'readme' }],
-}));
+const getDocuments = () =>
+  Object.keys(data)
+    .slice(1)
+    .map(docs =>
+      // @ts-ignore
+      data[docs].map(({ id, pagePath, children }) => ({
+        title: id,
+        pages: children
+          ? [...remapChild(children || [], id, 'nested-docs')]
+          : [{ id: 'readme', title: id, path: pagePath, type: 'readme' }],
+      })),
+    );
+
```

Comment:
This can also be more concise
`const newDocs: any[] = getDocuments().map(x => ...x);`